src/resultscallercalleepage.cpp

```diff
@@ -178,7 +178,8 @@ ResultsCallerCalleePage::toSourceMapLocation(const QString& location, const Data
 
 ResultsCallerCalleePage::SourceMapLocation ResultsCallerCalleePage::toSourceMapLocation(const QModelIndex& index) const
 {
-    Q_ASSERT(qobject_cast<const SourceMapModel*>(index.model()));
+    const auto proxy = qobject_cast<const QSortFilterProxyModel*>(index.model());
```

Comment:
the `proxy` is only used for the assertion, but nowhere else. I'd say we could/should remove the assert completely.

alternatively, make this:

```
Q_ASSERT([]() {
    const auto proxy = q_object_cast<const QAbstractProxyModel*>(index.model());
    return proxy && qobject_cast<const SourceMapModel*>(proxy->sourceModel());
}());
```

Reply:
I tried the lambda but it didn't work (could not capture index) so I removed the assertion